### DISCUSSION TITLE
Add a Sender field to the NotificationPayload Struct

### DISCUSSION THREAD

**Discussion by mizifih** [autobrr/autobrr#1722] [Ideas]
It would be awesome if the message came with an `<b>Autobrr</b>` field before everything else, which is already very neat. I’m not sure if this is common, but I use the same bot in Telegram for all my Arr notifications, and it would be even neater if the message included a Sender.

I know it's a small, maybe even petty request, but it would be great for us OCD nit-picking people. :)

I'm no coder, and I’m just being curious and nosy about the code, but after inspecting it a bit, are changes only needed in the `NotificationPayload` struct, `MessageBuilderPlainText{}` and `MessageBuilderHTML{}` functions to make it work? Did I get an _A+_? LOL

Thank you, guys, for this awesome software!

**Comment by zze0s** [comment]
Added in #1723

**Comment by Verwundert** [comment]
I can understand that some people use a single bot for everything, but others, like me, use multiple bots for different purposes.
Both sides have the right to set up their automation systems in the way they prefer, but this change affects people like me.
I would appreciate it if this new feature, which fortunately is only in development for now, could be manually adjusted.

**Reply by mizifih** [reply]
> I can understand that some people use a single bot for everything, but others, like me, use multiple bots for different purposes. Both sides have the right to set up their automation systems in the way they prefer, but this change affects people like me. I would appreciate it if this new feature, which fortunately is only in development for now, could be manually adjusted.

Added in https://github.com/autobrr/autobrr/pull/1723
